tslib/src/functions-throttled/school-import/data-access/school.data.service.ts

```diff
@@ -1,25 +1,23 @@
 import * as mssql from 'mssql'
 import { ILogger } from '../../../common/logger'
-import { ISchoolImportJobResult, SchoolImportJobResult } from '../SchoolImportJobResult'
+import { IJobOutput, SchoolImportJobOutput } from '../SchoolImportJobOutput'
 import { ISchoolRecord } from './ISchoolRecord'
 import { ISqlService, SqlService } from '../../../sql/sql.service'
-const name = 'school-import'
 
 export interface ISchoolDataService {
-  getJobId (): Promise<number | undefined>
-  updateJobStatus (jobId: number, jobStatusCode: string): Promise<any>
-  updateJobStatusWithResult (jobId: number, jobStatusCode: string, jobResult: ISchoolImportJobResult): Promise<any>
-  updateJobStatusWithResultAndError (jobId: number, jobStatusCode: string, jobResult: ISchoolImportJobResult, error: Error): Promise<any>
-  bulkUpload (schoolData: ISchoolRecord[]): Promise<SchoolImportJobResult>
+  // updateJobStatus (jobId: number, jobStatusCode: string): Promise<any>
```

Comment:
Is there a reason to comment these out rather than delete them?

Reply:
ah yes, those need tidying up now that the refactor is complete